docs/en/connector-v2/sink/AmazonSqs.md

```diff
@@ -0,0 +1,45 @@
+# AmazonSqs
+
+> Amazon SQS sink connector
+
+## Support Those Engines
+
+> Spark<br/>
+> Flink<br/>
+> SeaTunnel Zeta<br/>
+
+## Description
+
+Write data to Amazon SQS
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [x] [stream](../../concept/connector-v2-features.md)
+- [x] [exactly-once](../../concept/connector-v2-features.md)
+- [ ] [column projection](../../concept/connector-v2-features.md)
+- [ ] [parallelism](../../concept/connector-v2-features.md)
+- [ ] [support user-defined split](../../concept/connector-v2-features.md)
+
+## Options
```

Comment:
amazonsqs and SeaTunnel type mapping We also want to tell users that it is best to reflect this in the documentation

Reply:
the sqs is using message ( json/text) doesn't have a type, so we don't need type mapping i believe